org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss

```diff
@@ -148,13 +148,31 @@
     }
   }
 
+  .crm-core-form-recurringentity-block-ends {
+    input[type="radio"]:checked + label[for]:after {
+      top: 15px;
+    }
+  }
+
   .crm-core-form-recurringentity-block-absolute_date {
     position: relative;
 
     td {
       &.label {
         visibility: hidden;
       }
+
+      input[type="radio"]:checked + label[for]:after {
+        top: 15px;
+      }
+
+      .select2-container .select2-choice .select2-arrow b {
+        background: url("select2.png") no-repeat 0 1px !important;
```

Comment:
Are you sure about this line? 
I don't think there is a background image in the same directory of the minified css! 
Are you sure it doesn't export "Image not found" error in your browser console?